Sprint-1/exercises/random.js

```diff
@@ -1,7 +1,19 @@
 const minimum = 1;
 const maximum = 100;
 
-const num = Math.floor(Math.random() * (maximum - minimum + 1)) + minimum;
+const randomNum = Math.random();
+
+const num = Math.floor(randomNum * (maximum - minimum + 1)) + minimum;
+
+console.log(`Random = ${randomNum}`);
+console.log(`Rounder = ${maximum - minimum + 1}`);
+console.log(`Multiplied = ${randomNum * (maximum - minimum + 1)}`);
+console.log(`Rounded num = ${Math.floor(randomNum * (maximum - minimum + 1))}`);
+console.log(`num = ${num}`);
+
+/* I saw here getting a random number from 1 to 100, 
+to which the minimum number is added
+ */
```

Comment:
To test your understanding, how would you write an expression (without using any variable in the expression) that yields a random integer between -5 and 3 (including -5 and 3)?

Reply:
`Math.floor(Math.random() * 9) - 5`

`Math.floor(Math.random() * 9)` - getting random numbers from 0 to 8
Then, 0 - 5 = -5 and 8 - 5 = 3